solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,130 @@
+"""Simulating an ATM. """
+
+import sys
+
+ATM_FILE = 1
+DICTIONARY = {}
+
+
+def check_the_balance(costumer_id_st):
+    """Check the state of costumer's money in the bank.
+
+    Args:
+    costumer_id_st(string): costumer id.
+    """
+    balance = ''
+    for i in DICTIONARY.keys():
+        if costumer_id_st in i:
+            balance = DICTIONARY[i][1]
+    return balance
+
+
+def cash_withdrawal(costumer_id_st, amount_to_withdrawal):
+    """Withdrawal from the ATM amount of money.
+
+    Args:
+    costumer_id_st(string): costumer id.
+    amount_to_withdrawal(integer): amount the costumer wants to withdrawal.
+    """
+    balance_now = int(check_the_balance(costumer_id_st))
+    if balance_now >= amount_to_withdrawal:
+        new_balance = balance_now - amount_to_withdrawal
+        for i in DICTIONARY:
+            if costumer_id_st in i:
+                DICTIONARY[i][1] = str(new_balance)
+        return f"you have now: {new_balance}"
+    return f"You have now {balance_now}. " \
+           f"it's impossible to withdrawal more money than what you have"
+
+
+def cash_deposit(costumer_id_st, amount_to_deposit):
+    """deposit to the bank some of money.
+
+    Args:
+    costumer_id_st(string): costumer id.
+    amount_to_deposit(integer): amount the costumer wants to deposit.
+    """
+    if amount_to_deposit > 0:
+        balance_now = int(check_the_balance(costumer_id_st))
+        new_balance = balance_now + amount_to_deposit
+        for i in DICTIONARY:
+            if costumer_id_st in i:
+                DICTIONARY[i][1] = str(new_balance)
+        return f"you have now {new_balance}"
+    return "the amount is not good"
+
+
+def change_password(costumer_id_st, new_password):
```

Comment:
As a general comment, try not returning string messages from methods. If a method verify the cash amount in bank, return it as int and not as a message, the in the main program print whatever you want. If you want to execute a process and return whether it worked or not, return a bool or raise an exception if it didn't worked. 
Anyway don't return string messages from methods.